cmd/stomp-tool/main.go

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
The name of this tool should not contain `stomp`, as STOMP is just an implementation detail. Same applies to the source code: the word STOMP should not appear in the code of clients using this library.

Reply:
@jhernand BTW this PR does not work for me yet :-) just put is here so @zgalor and me can sync on this testing client.

Reply:
@jhernand taking parts of this PR that do work now into other PR's e.g. https://github.com/container-mgmt/messaging-library/pull/8

Reply:
@jhernand it's called `messaging-tool` now